### PR TITLE
Flake ref output

### PR DESCRIPTION
**Pull request by grahamc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
 
 ## Integration
 
-This action sets the `flakeref` output to the exact name and version that was published.
-The flake reference can be used in subsequent steps or workflows as part of a deployment process.
+This action sets outputs for integrating into CD pipelines:
```
Comment (lucperkins) [inline README.md:19]: Spell out continuous delivery

File: README.md [hunk 2]
```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
...
+
+- `flake_name` Name of the flake. Example: `DeterminateSystems/flakehub-push`
+- `flake_version` Name of the flake. Example: `0.1.99+rev-2075013a3f3544d45a96f4b35df4ed03cd53779c`
```
Comment (lucperkins) [inline README.md:22]: "Name of the flake" is repeated from above here

File: README.md (hunk 1 above)
Comment (lucperkins) [inline README.md:19]: Let's make this a table with rows for name, description, and example

**Review by lucperkins — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
...
+- `flakeref_descriptive` A loose reference to this release. Depending on this reference will require at least this version, and will also resolve to newer releases. This output is not sufficient for deployment pipelines, use `flake_exact` instead. Example: `DeterminateSystems/flakehub-push/0.1.99+rev-2075013a3f3544d45a96f4b35df4ed03cd53779c`
```
[the remark this answers is not in the document]
Comment (lucperkins) [reply README.md:23]: +1 for `flakeref_at_least`

File: README.md [hunk 4]
```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
...
+- `flake_exact` A precise reference that always resolves to this to this exact release. Example: `DeterminateSystems/flakehub-push/=0.1.99+rev-2075013a3f3544d45a96f4b35df4ed03cd53779c`
```
[the remark this answers is not in the document]
Comment (grahamc) [reply README.md]: Fixuped the docs!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (grahamc) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (grahamc) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (grahamc) [reply README.md]: Done!

**Review by lucperkins — APPROVED** [review]

**Review by cole-h — APPROVED** [review]

**Pull request merged**